src/components/Chart.js

```diff
@@ -1,5 +1,64 @@
-import React from "react";
+import React, { Component } from "react";
+import ChartComponent, { Line } from "react-chartjs-2";
 
-const Chart = () => <div>Chart Component</div>;
+let values = [
+  {
+    date: "some_date_1",
+    value: 1
+  },
+  {
+    date: "some_date_2",
+    value: 2
+  },
+  {
+    date: "some_date_3",
+    value: 3
+  }
+];
+
+class Chart extends Component {
```

Comment:
When you change date to real dates, then please check how to make this graph XY.
Tip. Date can be converted to numeral value.